src/main/java/org/alliancegenome/curation_api/jobs/util/CsvSchemaBuilder.java

```diff
@@ -0,0 +1,60 @@
+package org.alliancegenome.curation_api.jobs.util;
+
+import com.fasterxml.jackson.dataformat.csv.CsvSchema;
+
+public class CsvSchemaBuilder {
+
+	public static CsvSchema psiMiTabSchema() {
+		CsvSchema schema = CsvSchema.builder()
```

Comment:
Does the file have a header line? If so you wouldn't need this otherwise you do.

Reply:
It has multiple header lines - I couldn't see a way of specifying which header line was the header line to parse and which to ignore

Reply:
```
##########################################################################
#
# Data type: Molecular Interactions
# Data format: PSI-MI TAB 2.7 Format
# README: https://github.com/HUPO-PSI/miTab/blob/master/PSI-MITAB27Format.md
# Source: Alliance of Genome Resources (Alliance)
# Source URL: http://alliancegenome.org/downloads
# Help Desk: help@alliancegenome.org
# TaxonIDs: NCBI:txid9606, NCBI:txid10116, NCBI:txid10090, NCBI:txid8355, NCBI:txid8364, NCBI:txid8355, NCBI:txid8364, NCBI:txid7955, NCBI:txid7227, NCBI:txid6239, NCBI:txid559292, NCBI:txid2697049
# Species: Homo sapiens, Rattus norvegicus, Mus musculus, Xenopus laevis, Xenopus tropicalis, Xenopus laevis, Xenopus tropicalis, Danio rerio, Drosophila melanogaster, Caenorhabditis elegans, Saccharomyces cerevisiae, Severe acute respiratory syndrome coronavirus 2
# Alliance Database Version: 7.1.0
# File generated (UTC): 2024-02-28 16:32
#
##########################################################################
#ID(s) interactor A	ID(s) interactor B	Alt. ID(s) interactor A	Alt. ID(s) interactor B	Alias(es) interactor A	Alias(es) interactor B	Interaction detection method(s)	Publication 1st author(s)	Publication Identifier(s)	Taxid interactor A	Taxid interactor B	Interaction type(s)	Source database(s)	Interaction identifier(s)	Confidence value(s)	Expansion method(s)	Biological role(s) interactor A	Biological role(s) interactor B	Experimental role(s) interactor A	Experimental role(s) interactor B	Type(s) interactor A	Type(s) interactor B	Xref(s) interactor A	Xref(s) interactor B	Interaction Xref(s)	Annotation(s) interactor A	Annotation(s) interactor B	Interaction annotation(s)	Host organism(s)	Interaction parameter(s)	Creation date	Update date	Checksum(s) interactor A	Checksum(s) interactor B	Interaction Checksum(s)	Negative	Feature(s) interactor A	Feature(s) interactor B	Stoichiometry(s) interactor A	Stoichiometry(s) interactor B	Identification method participant A	Identification method participant B
```